### ISSUE TITLE
Suggestion: Add Mac Keys Icons

### ISSUE THREAD

**Issue by GustavoA95** [monsterbrain/keyboard-shortcut-image-generator#11]
Some Mac keys icons are not implemented.

⇧ Shift

⌃ Control

⌥ Option

**Comment by monsterbrain** [comment]
Let me clarify. Is it about adding the icons ?

**Comment by GustavoA95** [comment]
Yes. The idea would be to add the symbols for the Mac keyboard keys

![image](https://github.com/monsterbrain/keyboard-shortcut-image-generator/assets/83519205/ad47c7e9-1e24-4150-ad27-12aee142bdac)

I created a fork and started implementing it

![image](https://github.com/monsterbrain/keyboard-shortcut-image-generator/assets/83519205/2a3cac8a-67ef-4973-a8b6-42122dbedf76)

**Comment by monsterbrain** [comment]
Really Cool. Hope you put a PR after implementing it 👍

**Comment by GustavoA95** [comment]
I submitted a PR with the changes implemented
